lib/symengine.rb

```diff
@@ -33,6 +33,33 @@ def Function(n)
     def evalf(operand, prec=53, real=false)
         return _evalf(operand, prec, real)
     end
+    def lambdify(exp, syms)
+      eval(SymEngine::Utils::lambdify_code(exp, syms))
+    end
+  end
+  module Utils
+    class << self
+      def evalf_dirichlet_eta(exp)
+        SymEngine::evalf(SymEngine::dirichlet_eta(exp))
+      end
+      def evalf_zeta(exp)
+          SymEngine::evalf(SymEngine::zeta(exp))
+      end
+      def lambdify_code(exp, syms)
+        str = exp.to_s  
+        sym_map = syms.join(",")
+        str.gsub!(/[\d\.]+/, 'Rational(\0,1)')
```

Comment:
Are you sure it will cover all possible numbers in sane way?.. It looks a bit hack-ey.